src/geopmread_main.cpp

```diff
@@ -69,7 +69,9 @@ int main(int argc, char **argv)
                         "  DOMAIN_INDEX: index of the domain, starting from 0\n"
                         "\n"
                         "  -d, --domain                     print domain of a signal\n"
+                        "  -D, --domain-all                 print domains detected\n"
```

Comment:
Remove -d functionality to print domain of individual signals.  Use -i for that.

Move -D behavior to -d.